HandheldCompanion/Devices/Lenovo/LegionGo.cs

```diff
@@ -207,11 +207,13 @@ public LegionGo()
         DefaultLayout.ButtonLayout[ButtonFlags.B8] = new List<IActions>() { new MouseActions { MouseType = MouseActionsType.ScrollDown } };
 
         Init();
-
         Task<bool> task = Task.Run(async () => await GetFanFullSpeedAsync());
-        bool FanFullSpeed = task.Result;
+        bool FanFullSpeed;
```

Comment:
value is never used.. Also better to do .Wait() on Task. Checking task status on something async is pretty weird.

Reply:
I know, just commenting about it :) 
@Xirdneth